index.bs

```diff
@@ -3590,6 +3606,11 @@ Note: The {{CollectedClientData}} may be extended in the future. Therefore it's
     ::  This member contains the fully qualified [=origin=] of the requester, as provided to the authenticator by the client, in
         the syntax defined by [[!RFC6454]].
 
+    :   <dfn>topOrigin</dfn>
+    ::  This member contains the fully qualified [=top-level origin=] of the requester, as provided to the authenticator by the client, in
```

Comment:
Great point! Must admit I just copied the description from `origin`, and presumed that all of CollectedClientData was passed to the authenticator. That is not true of course 🤣 - only a hash of the JSON is, and `origin` is passed indirectly I believe (via `rpId`, which is checked against `origin` and the algorithm only continues if they match).

I'm happy to reword if you or @ve7jtb have thoughts here; perhaps just drop the authenticator mention and say:

`This member contains the fully qualified [=top-level origin=] of the requester, in the syntax defined by [[!RFC6454]]. If {{CollectedClientData/crossOrigin}} is `false`, this member should be identical to {{CollectedClientData/origin}}.`

?

Reply:
Would a client be expected to populate topOrigin in a traditional, top level call (no iframe)?

Reply:
Great question! As written, the PR does say a client should populate `topOrigin` in a traditional, top level call, and populate it identically to `origin`. However it might be reasonable to say that `topOrigin` is empty (or null?) in those situations. I don't think I have a preference in either direction.

Reply:
I think empty or not present makes sense for non-iframe scenarios, but required for iframe.

@akshayku @ve7jtb thoughts?

Reply:
Empty or not present in non-iframe scenario makes sense to me too. And it should be present in iframe scenarios.

Reply:
Thanks. I've updated the PR to be contingent on `sameOriginWithAncestors` being `false`. Note that this means `topOrigin` would not be set in the following scenario:

- relying-party.example
  - relying-party.example <-- a call from this sub-frame would not set `topOrigin`

If we would prefer it be set for *all* iframe scenarios, I can do that too.